drift/lib/src/web/channel_legacy.dart

```diff
@@ -1,4 +1,4 @@
-import 'dart:html';
+import 'dart:html'; // ignore: deprecated_member_use
```

Comment:
My first set of CI Tests failed because the dart analyzer complained about this import. I've specifically ignored it now.

Reply:
Ignoring is actually the best thing we can do here. This library is guarded behind conditional exports and only kept here for backwards compatibility.